functions/src/quiz/question-create.ts

```diff
@@ -16,7 +16,7 @@ export const QuizQuestionCreate = functions
             .collection('quizzes')
             .doc(context.params.quizId);
 
-        batch.update(gameRef, { status: 'READY' });
+        batch.update(gameRef, { status: 'OPEN' });
```

Comment:
I think this will effect the ability to start a new game in the quiz? need to double check, but it probably needs an update there to support this value

Reply:
I did a search and only saw `READY` in reference to a `player` not the game or quiz